gis_exercise_2

```diff
@@ -112,7 +112,7 @@ poly
 # Read the data into memory using Pandas
 import pandas as pd
 import numpy as np
-kruger_park = pd.read_csv('some_posts.txt', sep=',')
+kruger_park = pd.read_csv('data/some_posts.txt', sep=',')
```

Comment:
where is this data folder? How does github understand from where to source the data?

Reply:
I created the data folder and then added `some_posts.txt` to that folder